extra-web/palemoon/spec

```diff
@@ -1,4 +1,5 @@
-VER=28.13.0
-GITSRC="https://github.com/MoonchildProductions/Pale-Moon"
-GITCO="tags/${VER}_Release"
-CHKSUMS="SKIP"
+VER=28.15.0
+SRCS="tbl::https://repo.palemoon.org/MoonchildProductions/Pale-Moon/archive/${VER}_Release.tar.gz tbl::https://repo.palemoon.org/MoonchildProductions/UXP/archive/RELBASE_20201024.tar.gz"
```

Comment:
Why rename the file?

Reply:
Cloning the entire Git repository (5.3GiB) is way too big compared to downloading tarballs of a specific tag. They do provide tarballs.
Will revert this change if applicable.

Reply:
Please revert back to Git since it makes the build routine much more streamlined.